_posts/meat/2020-06-07-pork-larb.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Pork Larb
+category: meat
+tags: recipeish
+---
+
+Writing this up so that Buro can make it too. <3
+
+Ingredients
+
+* 1 pound ground pork
+* 1/2 cup - 1 cup white jasmine/sticky rice
+* 1-2 limes
+* fish sauce
+* few tablespoons grapeseed oil
+* 5-6 dried arbol chilis (optional)
+* 2-3 fresh thai chilis, sliced (optional)
+* 1 finely sliced shallot (optional)
+* 1/2 bunch green onions, chopped (optional)
+* 1/2 bunch cilantro, chopped (optional)
+* few sprigs mint, chopped (optional) 
+* lettuce / chard leaves for serving (optional)
+
+Tools
+
+* 1 wok / cast iron pan
+* 1 blade grinder
+
+Technique
+
+* In dry pan, toast rice until all grains are evenly light brown. Stir frequently over medium heat.
```

Comment:
Would it be possible to group or consolidate these instructions. It'll make it easier to follow, especially 'on mobile'.